README.md

```diff
@@ -83,6 +83,12 @@ After adding above, add the following dependency definitions.
 
 <!-- OPTIONAL -->
 <dependency>
+    <groupId>org.terasoluna.gfw</groupId>
+    <artifactId>terasoluna-gfw-codepoints</artifactId>
+</dependency>
```

Comment:
- [ ] In [guideline](http://terasolunaorg.github.io/guideline/5.1.0.RELEASE/ja/Overview/FrameworkStack.html#terasoluna-gfw-codepoints), codepoints component is shown `terasoluna-gfw-codepoints` only. Let to align to it.